lib/widgets/map_widget.dart

```diff
@@ -92,11 +92,13 @@ class _MapWidgetState extends State<MapWidget> {
         listen: false,
       ).changeCurrentLocation(_location.toLatLng());
     }
-    SizeConfig().init(context);
     final _completer = Provider.of<MapData>(context).getCompleter;
     final _buildings = Provider.of<BuildingsData>(context);
     final pos = Provider.of<UserLocation>(context);
-
+    if (sizeConfig) {
+      SizeConfig().init(context);
+      sizeConfig = false;
+    }
```

Comment:
Probably could just initialize SizeConfig in HomePage() instead.

Reply:
You're absolutely right. Haha